noxfile.py

```diff
@@ -123,11 +123,11 @@ def select_plugins(session, directory: str) -> List[Plugin]:
     as well as the user plugins selection (via the PLUGINS environment variable).
     """
     assert session.python is not None, "Session python version is not specified"
-    blacklist = [".isort.cfg", "examples"]
+    blocklist = [".isort.cfg", "examples"]
```

Comment:
if you want to not use the word blacklist you will have to come with a better alternative than this. these are not blocks, and there is nothing blocked here.